templates/checkout/_partials/steps/addresses.tpl

```diff
@@ -113,15 +113,15 @@
       {/if}
 
       <div class="mt-4">
-        <button class="btn btn-outline-primary btn-with-icon d-none d-md-inline-block me-2 js-back" data-step="checkout-personal-information-step">
+        <button class="btn btn-outline-primary btn-with-icon d-block d-md-inline-block me-2 w-full w-md-auto mb-3 mb-md-0 js-back" data-step="checkout-personal-information-step">
           <div class="material-icons">arrow_backward</div>
           {l s='Continue to Personal Information' d='Shop.Theme.Actions'}
         </button>
 
         {if !$form_has_continue_button}
             <button type="submit" class="btn btn-primary btn-with-icon d-block d-md-inline-block w-full w-md-auto continue" name="confirm-addresses" value="1">
```

Comment:
```suggestion
            <button type="submit" class="btn btn-primary btn-with-icon d-block d-md-inline-block w-full w-md-auto float-end continue" name="confirm-addresses" value="1">
```

Reply:
I would use flex, justify content space between. The floats sometime cause weird parent div height behavior. 👍